plone/app/z3cform/templates/checkbox_input.pt

```diff
@@ -7,113 +7,42 @@
       "
       tal:omit-tag=""
 >
-  <div tal:condition="python:len(items) &gt; 0"
+  <div id="${view/id}"
+       tal:condition="python:len(items) &gt; 0"
        tal:omit-tag="single_checkbox"
-       tal:attributes="
-         id view/id;
-       "
   >
     <div class="form-check"
+         id="${python:single_checkbox and view.id or None}"
          tal:repeat="item items"
-         tal:attributes="
-           id python:single_checkbox and view.id or None;
-         "
-    >
-      <input class=""
-             id=""
-             accesskey=""
-             alt=""
-             checked="checked"
-             name=""
-             tabindex=""
-             title=""
+    ><!--! NOTE:
+           we cannot autoset the attributes here with "view/attributes",
+           because the "id" would get overwritten with the field_id
+           instead of the correct sequence_item_id -->
+      <!--! NOTE #2:
+           Since most people import the "CheckBoxFieldWidget"
+           directly from z3c.form.browser.checkbox our adapter
+           which sets the Bootstrap "form-check-input" css class
+           isn't invoked. For this reason we manually add the
+           class here in the template.
+           When all imports are refactored to
+           "plone.app.z3cform.widgets.checkbox.CheckBoxFieldWidget"
+           this can be cleaned up. -->
```

Comment:
```suggestion
    ><tal:c condition="nothing">
    NOTE:
   Since most people import the `CheckBoxFieldWidget` directly from
   `z3c.form.browser.checkbox` our adapter which sets the Bootstrap
   `form-check-input` CSS class isn't invoked. For this reason we manually add
   the class here in the template. When all imports are refactored to
   `plone.app.z3cform.widgets.checkbox.CheckBoxFieldWidget` this can be cleaned
   up.
    </tal:c>
```